apps/vscode.py

```diff
@@ -0,0 +1,87 @@
+from talon.voice import Context, Key, press, Str
+from user.utils import parse_words_as_integer
+from user.utils import parse_words_as_integer, repeat_function, threeDigitNumber
+
+context = Context('VSCode', bundle='com.microsoft.VSCode')
+
+def jump_to_line(m):
+    line_number = parse_words_as_integer(m._words[1:])
+
+    if line_number == None:
+        return
+
+    # Zeroth line should go to first line
+    if line_number == 0:
+        line_number = 1
+
+    press('cmd-g')
+    Str(str(line_number))(None)
+    press('enter')
+
+def jump_tabs(m):
+    line_number = parse_words_as_integer(m._words[1:])
+
+    if line_number == None:
+        return
+
+    for i in range(0, line_number):
+        press('cmd-alt-right')
+
+def jump_to_next_word_instance(m):
+    press('escape')
+    press('cmd-f')
+    Str(' '.join([str(s) for s in m.dgndictation[0]._words]))(None)
+    press('return')
+
+def select_lines_function(m):
+    divider = 0
+    for word in m._words:
+        if str(word) == 'until':
+            break
+        divider += 1
+    line_number_from = int(str(parse_words_as_integer(m._words[2:divider])))
+    line_number_until = int(str(parse_words_as_integer(m._words[divider+1:])))
+    number_of_lines = line_number_until - line_number_from
+
+    press('cmd-g')
+    Str(str(line_number_from))(None)
+    press('enter')
+    for i in range(0, number_of_lines+1):
+        press('shift-down')
+
+context.keymap({
+    # Selecting text
+    'select line' + threeDigitNumber + 'until' + threeDigitNumber: select_lines_function,
+
+    # Finding text
+    'find': Key('cmd-f'),
+    'find next <dgndictation>': jump_to_next_word_instance,
+
+    # Clipboard
+    'clone': Key('alt-shift-down'),
+    
+    # Navigation
+    'line' + threeDigitNumber: jump_to_line,
+    'Go to line': Key('cmd-g'),
```

Comment:
if you make the three digit number optional, you can combine this into a single command. Check out `utils.optional_numerals`

Reply:
sorry completely forgot that I had this in another file where it was optional already, but I used yours instead. Don't know what you mean by one command though